File: alwin-middleware-grapescode/alwin-core-api/src/main/java/com/codersteam/alwin/core/api/model/activity/IssueTypeWithDefaultActivities.java
package com.codersteam.alwin.core.api.model.activity;

import com.codersteam.alwin.core.api.model.issue.IssueTypeDto;

import java.util.List;

/**
 * @author Dariusz Rackowski
 */
public class IssueTypeWithDefaultActivities {

    private IssueTypeDto issueType;

    private List<DefaultIssueActivityDto> defaultActivities;

    public IssueTypeWithDefaultActivities() {
    }

    public IssueTypeWithDefaultActivities(final IssueTypeDto issueType, final List<DefaultIssueActivityDto> defaultActivities) {
        this.issueType = issueType;
        this.defaultActivities = defaultActivities;
    }

    public IssueTypeDto getIssueType() {
        return issueType;
    }

    public void setIssueType(final IssueTypeDto issueType) {
        this.issueType = issueType;
    }

    public List<DefaultIssueActivityDto> getDefaultActivities() {
        return defaultActivities;
    }

    public void setDefaultActivities(final List<DefaultIssueActivityDto> defaultActivities) {
        this.defaultActivities = defaultActivities;
    }
}

File: alwin-web-grapescode/src/app/activity/issue-type-with-default-activities.ts
import {IssueType} from '../issues/shared/issue-type';
import {DefaultIssueActivity} from './default-issue-activity';

export class IssueTypeWithDefaultActivities {
  issueType: IssueType;
  defaultActivities: DefaultIssueActivity[];
  /*
   *  Ustawia issueType na podstawie już istniejącego
   */
  setIssueType(issueType: IssueType) {
    this.issueType = issueType;
  }
}
